drupal-issue-tags.js

```diff
@@ -0,0 +1,53 @@
+(async () => {
+  const src = chrome.runtime.getURL("common.js");
+  const { utils } = await import(src);
+
+  var url = document.URL;
+  const regex1 = /https:\/\/www\.drupal\.org\/project\/.*\/issues\/.*/g;
+  const regex2 = /https:\/\/www\.drupal\.org\/node\/add\/.*/g;
+  if (url.match(regex1) || url.match(regex2)) {
+    const assignedInput = document.getElementById("edit-taxonomy-vocabulary-9-und");
+    const assignedText = assignedInput.value;
+    const existingTags = assignedText.split(", ");
+
+    const issueTagsDiv = document.createElement("div");
+    issueTagsDiv.className = "issueTagsDiv";
+    let commonIssueTags = [
+      'Needs tests',
+      'Needs issue summary update',
+      'Accessibility',
+      'core-mvp',
+    ]
+    commonIssueTags.forEach((tag)=> {
+      const tagDiv = document.createElement("button");
+      tagDiv.className = "tagDiv";
+      tagDiv.innerText = `${tag}`;
+      tagDiv.id = `${tag}`;
```

Comment:
This seem to work as an id for our purpose but this create an element with a invalid id format see 

https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/id

I would suggest making function `createIdForTag(tag)` which convert it to a valid id value and also a  prefix incase we have existing element with id somewhere that already uses the tag id. so it would return something  like `tag-button-core-mvp`

Of course that would mean you could not use `document.getElementById(`${tag}`)` anymore. But you could make another function that would basically be something like

```
function getElementForTag(tag) {
      return document.getElementById(createIdForTag(tag));
    }
```

Then you can just use `getElementForTag(tag)` everywhere you had `document.getElementById(`${tag}`)`

Reply:
forgot to add this example of how to convert to a valid id https://stackoverflow.com/questions/25317997/function-to-create-a-valid-id-from-string-regular-expression